fla/layers/mom_varlen.py

```
@@ -0,0 +1,724 @@
+        else:
+            k = torch.stack([k_expert(hidden_states[i]) for i, k_expert in enumerate(self.k_proj)], dim=0)
+            v = torch.stack([v_expert(hidden_states[i]) for i, v_expert in enumerate(self.v_proj)], dim=0)
+            beta = torch.stack([b_expert(hidden_states[i]).sigmoid() for i, b_expert in enumerate(self.b_proj)], dim=0)
+            g = torch.stack([-self.A_log.float().exp() * F.softplus(a_expert(hidden_states[i]).float() + self.dt_bias) for i, a_expert in enumerate(self.a_proj)], dim=0)
+
+        if self.use_short_conv:
+            conv_state_q, conv_state_k, conv_state_v = [None, None], [None, None], [None, None]
+            if last_state is not None:
+                conv_state_q, conv_state_k, conv_state_v = last_state['conv_state']
+            conv_mask = attention_mask[:, -hidden_states.shape[2]:].repeat_interleave(self.num_memories, 0) if attention_mask is not None else None
+            seq_idx=kwargs.get('seq_idx', None)
```

Ruff (0.11.9)

Use `kwargs.get('seq_idx')` instead of `kwargs.get('seq_idx', None)`

Replace `kwargs.get('seq_idx', None)` with `kwargs.get('seq_idx')`

(SIM910)